client/components/svg-icon/index.tsx

```diff
@@ -9,7 +9,9 @@ interface Props {
 	size?: number;
 }
 
-function SVGIcon( props: Assign< React.SVGProps< SVGSVGElement >, Props > ) {
+type AllProps = Assign< React.SVGProps< SVGSVGElement >, Props >;
+
+const SVGIcon = React.forwardRef< SVGSVGElement, AllProps >( ( props: AllProps, ref ) => {
```

Comment:
The changes in this file copy [what's already done in GridIcon](https://github.com/Automattic/wp-calypso/blob/e4686f960553f22b64fdce126f4f7aa946467e2d/packages/components/src/gridicon/index.tsx) to allow the component to receive a ref prop.

This is needed to trigger the popup for follow settings when clicking the icon.